src/Profiler.php

```diff
@@ -123,9 +123,9 @@ public function setEvent(EventInterface $event)
     /**
      * Returns the profiler event object.
      *
-     * @return self
+     * @return ProfilerEvent
      */
-    public function getEvent()
+    public function getEvent(): ProfilerEvent
```

Comment:
Please do not add native php typehints in this PR as this introduces a BC break. 
Focus on PHP 8.0 compatibility only.
Thanks!